src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_InvalidExpression.cs

```diff
@@ -253,9 +259,8 @@ void F()
 }
 ";
             string expectedOperationTree = @"
-IIncrementOrDecrementExpression (UnaryOperandKind.Invalid) (BinaryOperationKind.Invalid) (OperationKind.IncrementExpression, Type: System.Object, IsInvalid) (Syntax: '++x')
-  Left: ILocalReferenceExpression: x (OperationKind.LocalReferenceExpression, Type: Program) (Syntax: 'x')
-  Right: ILiteralExpression (OperationKind.LiteralExpression, Type: System.Object, Constant: 1) (Syntax: '++x')
+IIncrementOrDecrementExpression (Prefix) (OperationKind.IncrementExpression, Type: System.Object, IsInvalid) (Syntax: '++x')
```

Comment:
>System.Object [](start = 83, length = 13)

I am not sure why the type is Object in this case, we should investigate.

Reply:
tagging @mavasani it looks like it did increment expression work? should I open a bug?

Reply:
Yes, please open an issue.

Reply:
done - https://github.com/dotnet/roslyn/issues/22554